migrations/fetch-program.js

```diff
@@ -297,20 +315,105 @@ const parseTerm = (term, type, locale) => {
   return undefined;
 };
 
-const parseTag = (tag, type, locale) => {
-  if (tag) {
-    return {
-      type,
-      slug: tag.slug,
-      locales: {
-        [locale]: {
-          name: tag.name,
-          slug: tag.slug,
-        },
+const parseTags = async (tags, type, locale) => {
+  if (!tags) return undefined;
+
+  const results = [];
+  for (const tag of tags) {
+    const result = await parseTag(tag, type, locale);
```

Comment:
Is here possibility to get array with undefined values? Since parseTag returns undefined, but before pushing to results array we dont do any kind of null checks. Does array with undefined values do some harm?

Reply:
Good point. I don't believe it is possible, but Strapi doesn't seem to like it if we pass undefined values in relation arrays, so I'll add a check here.